src/avram/virtual_operation.cr

```diff
@@ -31,4 +31,8 @@ class Avram::VirtualOperation
   def attributes
     virtual_attributes
   end
+
+  def self.param_key
+    name.underscore
```

Comment:
This is a little trickier. Since  `VirtualForm`  does not have a model to work with it is hard to tell what this should be. 

Should we underscore the operation name like I did here? `SignInUser` -> `sign_in_user`, `ResetPassword` -> `reset_password`

Or should we not even try to guess it and have people set a param_key when using a VirtualOperation? 

I think underscoring the class name is fine but would love opinions @jwoertink @edwardloveall

Reply:
I think default of underscored operation class name is fine, with a way to override it maybe?

Reply:
You can override now with `param_key :my_custom_key`! 👍

Reply:
Yeah, you can always override it anyway. Path of least resistance.

Reply:
Thanks for the input both of you! I’m glad we all agree. I’ll merge this in!